components/DumbComponent.js

```diff
@@ -0,0 +1,24 @@
+const React = require('react');
+const { Component } = require('react');
+
+class DumbComponent extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      mood: 'happy'
+    };
+    this.handleClick = this.handleClick.bind(this);
```

Comment:
I feel like it's cleaner to have an extra line between defining `this.state` and binding the methods. Total nitpick, though.